src/components/progress/progress.scss

```diff
@@ -0,0 +1,149 @@
+@import "../../themes/ionic.globals";
+
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:               rgb(16, 108, 2001) !default;
```

Comment:
Could you update this and then after some testing we could get it merged? Is it ready?